src/content.js

```diff
@@ -1,192 +0,0 @@
-/**
- * All of the items in the NugetFunctionality namespace indicate that the functions come from the Nuget Gallery JavaScript.
```

Comment:
> I specifically mirrored the Nuget site's scripts for parity in case their way of applying the tabs changed in the future. This loses some of that mirroring. I agree it's much cleaner / better script overall, but I was specifically going for parity with the original site.

That is true, there is no more Razor: [source](https://github.com/NuGet/NuGetGallery/blob/master/src/NuGetGallery/Views/Packages/DisplayPackage.cshtml#L40) to javascript: [source](https://github.com/SeanKilleen/paket-attacket/pull/10/files/ecfcc26a4846a7a7bf48c8a4c0f0fde173a67657#diff-95ffcbff3c841ad0a802793cabd8940eL8) mirroring but parity still exists by means of jQuery manipulation of existing tabs: [source](https://github.com/SeanKilleen/paket-attacket/pull/10/files/ecfcc26a4846a7a7bf48c8a4c0f0fde173a67657#diff-2b4ca49d4bb0a774c4d4c1672d7aa781R20)
One can argue that it is harder to track changes without exact code mirroring but I took different approach because copying templates can fail silently. You may end up in situation where you are suddenly out of sync but code won't fail, you'll still get element generated and injected into DOM but it won't be functional anymore. With jQuery approach chances are bigger that it will spit out nice big error stack trace once you try modifying something that isn't inside cloned fragment.

In terms of required effort I don't see significant difference between templates and jQuery manipulation because target fragments are simply small enough.

That being said I can turn it back to your original approach without changing the rest if that seems better to you?